tasks/main.yml

```diff
@@ -64,8 +64,11 @@
       paths:
         - "{{ role_path }}/vars"
 
-# "expect" is used downstream to automate creation of Nessus users
-- name: Install expect package
+# "expect" and "jq" are used downstream to automate creation of Nessus users
+- name: Install expect and jq packages
   package:
-    name: "{{ expect_package_name }}"
+    name: "{{ item }}"
     state: present
+  loop:
```

Comment:
No need for a loop here. package can take a list for name. Also, why not create a single variable for both packages?

Reply:
Right- forgot that we can do that.  See https://github.com/cisagov/ansible-role-nessus/pull/11/commits/2cd00b99eb6c8052f957757186fd87f89785ca27 - thanks!